cts/scheduler/scores/colocation-vs-stickiness.scores

```diff
@@ -57,7 +57,7 @@ pcmk__primitive_assign: FencingPass allocation score on rhel7-5: INFINITY
 pcmk__primitive_assign: dummy1 allocation score on rhel7-1: -INFINITY
 pcmk__primitive_assign: dummy1 allocation score on rhel7-2: -INFINITY
 pcmk__primitive_assign: dummy1 allocation score on rhel7-3: -INFINITY
-pcmk__primitive_assign: dummy1 allocation score on rhel7-4: -500
+pcmk__primitive_assign: dummy1 allocation score on rhel7-4: 0
```

Comment:
How did we lose the anti-colocation score? It should lose the battle compared to the stickiness in this test but I wonder if there are cases where it would make a difference.

Reply:
At the start, both group1 and group2 are active on rhel7-5. Not sure why that score ever got applied to 7-4 :)

Reply:
I ran `crm_simulate` on this input file. First `pcmk__group_assign()` runs on `group1` and its members and then `group2` and its members (probably correct although I would've expected `group2` first since it's the primary resource... must not matter for group_assign).

Then `pcmk__primitive_assign()` runs on each of `group2`'s members, and then each of `group1`'s members.

All the `pcmk__group_assign()` outputs had `INFINITY` for the current node `rhel7-5` and zeroes elsewhere. The `dummy2` resources got assigned first and chose `rhel7-4` due to anti-colocation with `dummy1`. Specifically, `dummy1` has an `INFINITY` preference for `rhel7-5` due to stickiness, so `-1 * INFINITY = -INFINITY`.

As a result, `dummy2` chose the other available node, `rhel7-4`. 

When `dummy1` resources got assigned, they anti-colocated with a score of -500.

After the fix, `dummy2` appropriately weights `dummy1`'s preference, so it gets a temporary score of `-0.0005 * INFINITY = -500` for `rhel7-5`. When its own preference of `INFINITY` is added, the result is `INFINITY`, and it chooses `rhel7-5` instead of incorrectly choosing `rhel7-4`.

Basically same thing then happens when assigning `dummy1`. It starts with an `INFINITY` score for `rhel7-5` due to stickiness. 500 gets subtracted from the score three times (for `dummy2a`, `dummy2b`, and `dummy2c`), but `INFINITY - 500 = INFINITY`.